imsv-docs-astro/src/content/networks/algorand-testnet.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Algorand Testnet
```

Comment:
I thought there were multiple testnets. Are we only supporting the main one?

Reply:
The algorand testnet is literally called "testnet". They also have "betanet" which we won't support at this stage.